packages/shared/components/popups/ActivityDetailsPopup.svelte

```diff
@@ -46,7 +46,7 @@
     })
 
     function handleExplorerClick(): void {
-        Platform.openUrl(`${explorerUrl}/block/${activity.id}`)
+        Platform.openUrl(`${explorerUrl}/block/${activity.transactionId}`)
```

Comment:
transactionId is optional value on the activity class? should it be non optional or should we handle the case where we don't have the transaction id?

Reply:
but yes, if we have no transactionId (e.g. if the transaction got pruned) we need to handle it